mobile/Ihc/components/MedicationInventory.js

```diff
@@ -90,6 +99,19 @@ export default class MedicationInventory extends Component<{}> {
     }
   }
 
+  getMedicationValues(medication) {
```

Comment:
Is this actually different than just the medication object itself? If so, please leave a comment describing how its different and how/why this is needed

Reply:
yup will do